types/genesis_test.go

```diff
@@ -48,6 +48,21 @@ func TestGenesisBad(t *testing.T) {
 				`},"power":"10","name":""}` +
 				`]}`,
 		),
+		// unsupported validator pubkey type
+		[]byte(
+			`{
+				"validators": [{
+					"pub_key":{"type":"tendermint/PubKeyEd25519","value":"AT/+aaL1eB0477Mud9JMm8Sh8BIvOYlPGC9KkIUmFaE="},
+					"power":"10",
+					"name":""
+				}],
+				"consensus_params": {
+					"validator": {"pub_key_types":["secp256k1"]},
+					"block": {"max_bytes": "100"},
+					"evidence": {"max_age_num_blocks": "100", "max_age_duration": "10"},
+				}
+			}`,
+		),
 	}
 
 	for _, testCase := range testCases {
```

Comment:
Do you mind improving the error message in line 70?

Reply:
Just saw your message. I pushed a commit with an error message that better reflects the test's purpose.